packages/jupyter-ai-magics/jupyter_ai_magics/__init__.py

```diff
@@ -1,34 +1,72 @@
+from langchain_core._import_utils import import_attr
+
```

Comment:
This looks like a private module, which means it may break in patch releases. Can we re-implement this in `jupyter_ai_magics` to prevent that in advance?

Reply:
Yes, that's the plan! I took a shortcut for PoC.